src/plugins/data_source/common/credentials/types.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectAttributes } from 'src/core/types';
+
+/**
+ * The scope of each credential is defined by the authentication type.
+ * It is currently shared by default within the tenant.
+ */
+export enum AuthType {
+  SharedAuthType = 'shared',
+}
```

Comment:
let's remove this for now, and add it when we implement private/shared credential

Reply:
removed